src/Chapter09/Listing09.21.ImplementStronglyTypedEquals.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_21
+{
+    public class Foo : IEquality<Foo>
```

Comment:
Do the examples use `IEquality<T>`? I would have expected them to use the built in `IEquatable<T>`

Reply:
Same as above!